src/renderer/components/+network-ingresses/ingress-class-set-default.injectable.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import type { IngressClass } from "../../../common/k8s-api/endpoints/ingress-class.api";
+
+// TODO: figure out if this needs to be injectable
+export const ingressClassSetDefaultInjectable = getInjectable({
```

Comment:
This is not used anywhere, so remove?

Reply:
It was preparation.. Please check again.